doc/kubernetes/modules/ROOT/pages/util/kc-failover.adoc

```diff
@@ -1,88 +1,65 @@
 = Simulate Keycloak Site Failover
-:description: How to automate the simulation of Keycloak site failure.
+:description: How to automate the simulation of Keycloak site failures.
 
 == Prerequisites:
 
-* A Keycloak instance replicated across two Openshift clusters with Infinispan xsite and an Aurora DB
-* Realm, user and client exist with the values required by the benchmark CLI command
+* A Keycloak instance replicated across two ROSA clusters with Infinispan xsite and an Aurora DB
+* xref:prerequisite/prerequisite-task.adoc[Task]
 
 == Running the failure test from the CLI
 
-=== Preparations
 
-* Extract the `+keycloak-benchmark-${version}.[zip|tar.gz]+` file
-* xref:benchmark-guide::preparing-keycloak.adoc[]
-* Make sure your local KUBECONFIG is set to the Openshift cluster which you want to fail.
+== Installation steps
 
-=== Parameters
+. Check out the Git repository from https://github.com/keycloak/keycloak-benchmark/.
 
-The failover scripts requires the following env variables to be set; `FAILOVER_MODE` and `DOMAIN`.
+. Change to the folder `link:{github-files}/provision/rosa-cross-dc/[provision/rosa-cross-dc]`.
+
+. Provide the <<required-variables>> in the `.env` file in the folder.
 
-The `FAILOVER_MODE` determines the type of failover that is initiated by the script and can be one of the following values:
-[cols='1,3']
-|===
-| FAILOVER_MODE | Description
 
-| [.nowrap]`HEALTH_PROBE`
-| Deletes the Keycloak aws-health-route so that Route53 will eventually failover.
+== Split Brain
 
-| [.nowrap]`KEYCLOAK_ROUTES`
-| Deletes all Keycloak routes so that Route53 will eventually failover, but requests to the old DNS IP addresses will fail.
-The Keycloak Operator is scaled down to 0 pods to prevent the Keycloak Ingress from being recreated.
+Two tasks are provided for split-brain failover, one to cause a split-brain and the other to recover from it:
 
-| [.nowrap]`CLUSTER_FAIL`
-| Deletes all Keycloak and Infinispan pods with no grace period and remove the associated StatefulSet. Both operators are
-scaled down to prevent the removed resources from being recreated.
+[source]
+----
+* split-heal:                             Heal a split-brain simulation by reenabling GossipRouters and reseting the global accelerator endpoints
+* split-trigger:                          Disables Infinispan GossipRouters on each cluster to simulate a split-brain scenario
+----
 
-| [.nowrap]`GOSSIP_ROUTER_FAIL`
-| Deletes the Infinispan Gossip Router pod with no grace period and removes the associated Deployment. The Infinispan
-operator is scaled down to prevent the removed resources from being recreated.
 |===
+|Variable |Details
 
-See below for a description of the other environment variables that can be configured.
+|ACCELERATOR_NAME
```

Comment:
Do we want to share an example of the load balancer nomenclature? For example, should we provide https vs not providing it?

Reply:
I'm not sure what you mean?

Reply:
I added this comment when I thought this was the URL, but after I looked at the code, it was the name itself, so I guess we don't need to elaborate on further.